pkg/cheatsheet/generate_test.go

```diff
@@ -27,6 +27,7 @@ func TestGetBindingSections(t *testing.T) {
 				{
 					ViewName:    "files",
 					Description: "stage file",
+					Key:         'a',
```

Comment:
the existing tests didn't specify a key so they all broke. Because I'm lazy I'm just using key 'a' everywhere to get the passing again